client-config/src/main/java/com/palantir/conjure/java/client/config/ClientConfigurations.java

```diff
@@ -44,7 +46,7 @@
     private static final Duration DEFAULT_WRITE_TIMEOUT = Duration.ofMinutes(5);
     private static final Duration DEFAULT_BACKOFF_SLOT_SIZE = Duration.ofMillis(250);
     private static final Duration DEFAULT_FAILED_URL_COOLDOWN = Duration.ZERO;
-    private static final boolean DEFAULT_ENABLE_GCM_CIPHERS = false;
+    private static final boolean DEFAULT_ENABLE_GCM_CIPHERS = !isJava8();
```

Comment:
Ideally, it would be great to provide a little comment and link here showing how GCM cipher suite performance got a lot better in java9, otherwise future readers might not remember why we special-cased java8 here